src/OnboardingSPA/components/NewfoldInterfaceSkeleton/index.js

```diff
@@ -9,23 +9,26 @@ import classnames from 'classnames';
 /**
  * WordPress dependencies
  */
-import { forwardRef, useEffect } from '@wordpress/element';
+import { forwardRef, useEffect, useContext } from '@wordpress/element';
 // eslint-disable-next-line  @wordpress/no-unsafe-wp-apis
 import { __unstableUseNavigateRegions as useNavigateRegions } from '@wordpress/components';
 import { __ } from '@wordpress/i18n';
 import { useMergeRefs } from '@wordpress/compose';
-// import bgAiImg from '../../static/images/ai_bg.png';
+// Needs to explicitly imported to be added dynamically
+import AIImg from '../../static/images/sitegen/ai_bg.png';
+import { ThemeContext } from '../ThemeContextProvider';
+import { THEME_DARK } from '../../../constants';
 
-function useHTMLClass( className ) {
+function useHTMLClass( className, isDarkMode ) {
 	useEffect( () => {
 		// eslint-disable-next-line no-undef
-		// const mainImage = new Image();
-		// mainImage.src = bgAiImg;
-		// mainImage.onload = () => {
-		// 	document.querySelector(
-		// 		'.nfd-onboarding-skeleton--sitegen'
-		// 	).style.backgroundImage = `url('${ bgAiImg }')`;
-		// };
+		const mainImage = new Image();
+		mainImage.src = AIImg;
+		mainImage.onload = () => {
+			document.querySelector(
+				'.nfd-onboarding-skeleton--sitegen'
+			).style.background = isDarkMode ? `url('${ AIImg }')` : '#ededed';
```

Comment:
Should we ensure that the element exists before accessing the `style` property? Should we remove the hardcoded `#ededed` and replace with a variable?

Reply:
Fixed.